library/src/lib/calendar/calendar2/calendar2-views/calendar2-month-view/calendar2-month-view.component.ts

```diff
@@ -1,17 +1,104 @@
-import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { Component, Input, Output, ViewEncapsulation, EventEmitter, ElementRef } from '@angular/core';
 
 @Component({
     selector: 'fd-calendar2-month-view',
     templateUrl: './calendar2-month-view.component.html',
     styleUrls: ['./calendar2-month-view.component.scss'],
     encapsulation: ViewEncapsulation.None
 })
-export class Calendar2MonthViewComponent implements OnInit {
+export class Calendar2MonthViewComponent {
+    /** @hidden The id of the calendar passed from the parent component. */
+    @Input() id: string;
 
-    constructor() {
+    /** @hidden A list of month names */
+    @Input() monthNames: string[];
```

Comment:
You should do 

```
@Input()
variable: type;
```

everywhere, but some places I see `Input() variable: type;`